### ISSUE TITLE
Symbol Icon Override

### ISSUE THREAD

**Issue by NickToye** [keremciu/sketch-iconfont#106]
Hi,

I have created a symbol which has an Icon in it from your plugin.  When I insert a symbol I'd like to be able to change the icon in the override section, but all I see is a box with a ? in it.  The base symbol has the icon in it no problem, just can't override.

**Comment by keremciu** [comment]
Hello @NickToye 

I'm trying to publish new version of the plugin, it will contains ability of changing symbol icon feature

I will let you know about new release

**Comment by NickToye** [comment]
Look forward to it.

**Comment by rvmourik** [comment]
Is this already possible?

**Comment by keremciu** [comment]
@NickToye @rvmourik yes, there is a possibility, nowadays I started to use in my daily job while trying to build a style-guide. I hope I can share a new plugin totally different from beginning to end, it will be more easy to use for style-guides.

**Comment by rvmourik** [comment]
Hi @keremciu 

That's nice to here, can you quickly tell what the current solution is until there is a new version of the plugin?

**Comment by amberRrucker** [comment]
@keremciu also looking for this solution

**Comment by jferracini** [comment]
also looking for this solution [2]

**Comment by keremciu** [comment]
@jferracini @amberRrucker @rvmourik 

sorry guys about not having time for this plugin in a long run but my personal issues(moving to berlin)  will be done soon then I could handle more issues here but right now I look like only maintainer to fix this issues

**Comment by jferracini** [comment]
Don't worry! Thanks for answer!

**Comment by LauraLalune** [comment]
+1 for this issue. @keremciu maybe it would be enough to be able to copy the unicode on the icon browser window. In this case one could choose IF to click on the icon and insert it, OR copy the unicode and paste into the override - just a quick workaround in case it is difficult to make it "speak" directly to the overrides :) hope you'll be on that soon. Your plugin is awesome!
